docs/css/buttons.css

```diff
@@ -0,0 +1,49 @@
+a.xournalppButton {
```

Comment:
`.download-button` is a better name for this.

Reply:
Thanks for naming suggestion, although I am not sure if `.download-button` is a proper name as I have used the same style in the `index.md` where buttons are used as quick links. That is why I have named them `xournalppButton` so that they could be used in other places than just download center and the naming would still make sense.